### PR TITLE
Event Single Re-skin - Organizer Block [TEC-3989]

### PR DESCRIPTION
**Pull request by EugeneKyale**
This PR contains some styles for the Organizer block reskin. 

Artifact: https://loom.com/share/1b460fc74c574708a6f1f3f6b2e6f0be
Ticket : https://theeventscalendar.atlassian.net/browse/TEC-3989

FYI, I'll add a change log to the final ticket addressing the re-skin. 
Also I confirmed that the Divi's zip contains the plugin name beginning with uppercase D. We can also confirm this on some of their demo sites [here](https://www.elegantthemes.com/gallery/divi/#layouts).

### CODE REVIEW

File: src/resources/postcss/tribe-events-full.pcss [hunk 1]
```diff
@@ -194,11 +194,13 @@ a.tribe-button,
...
 #tribe-events,
 #tribe-events-pg-template,
-.tribe-events-pg-template {
+.tribe-events-pg-template,
+#top.theme-enfold #tribe-events-pg-template {
 	padding: 40px 20px 20px;
 }
 
 #tribe-events-pg-template,
+#top.theme-enfold #tribe-events-pg-template,
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-full.pcss:203]: Curious why we're changing the page template here for enfold users. Isn't the goal here to only reskin the organizer block?

File: src/styles/event-organizer/frontend.pcss [hunk 2]
```diff
@@ -6,20 +6,58 @@
 .tribe-block__organizer__details {
 	margin: 15px 0;
 	position: relative;
+	border-top: 1px solid #d5d5d5;
+	padding: 32px 0;
```
Comment (paulmskim) [inline src/styles/event-organizer/frontend.pcss:10]: style order

File: src/styles/event-organizer/frontend.pcss [hunk 3]
```diff
@@ -6,20 +6,58 @@
...
+
+	@media screen and (max-width: 480px) {
+		padding: 22.5px 0;
+	}
```
Comment (paulmskim) [inline src/styles/event-organizer/frontend.pcss:14]: Mobile first. Also, 480px is a small breakpoint, maybe aim for something like 768px?

File: src/styles/event-organizer/frontend.pcss [hunk 4]
```diff
@@ -6,20 +6,58 @@
...
 	h3 {
-		font-weight: bold;
-		font-size: 1.3125rem;
-		color: #000;
+		font-weight: normal;
+		font-size: 1rem;
 		margin-bottom: 4px;
 		margin-top: 0;
+		line-height: 1.64;
+		letter-spacing: normal;
```
Comment (paulmskim) [inline src/styles/event-organizer/frontend.pcss:22]: style order

File: src/styles/event-organizer/frontend.pcss [hunk 5]
```diff
@@ -6,20 +6,58 @@
...
 
 	p {
 		font-size: 1rem;
-		color: #545D66;
+		color: #141827;
 		margin-bottom: 2px;
 		margin-top: 0;
 		font-weight: normal;
+		line-height: 1.64;
+		letter-spacing: normal;
```
Comment (paulmskim) [inline src/styles/event-organizer/frontend.pcss:36]: style order

**Review by paulmskim — CHANGES_REQUESTED** [review]

File: src/styles/event-organizer/frontend.pcss [hunk 6]
```diff
@@ -4,22 +4,76 @@
...
+.theme-Divi .tribe-block__organizer__details h3,
+.theme-Divi .tribe-block__organizer__details p {
+	padding-bottom: 0;
+}
+
+/* Genesis */
+.theme-genesis .tribe-block__organizer__details h3,
+.theme-genesis .tribe-block__organizer__details p {
+	font-size: 1.65rem;
+}
```
Comment (paulmskim) [inline src/styles/event-organizer/frontend.pcss:65]: let's nest these so we aren't repeating unnecessarily

**Comment by EugeneKyale** [comment]
I've implemented the nesting.

File: src/resources/postcss/tribe-events-full.pcss (hunk 1 above)
Comment (paulmskim) [reply src/resources/postcss/tribe-events-full.pcss:203]: @EugeneKyale any comment on this?

File: src/styles/event-organizer/frontend.pcss [hunk 7]
```diff
@@ -4,22 +4,85 @@
...
+
+/* Genesis */
+.theme-genesis {
+	.tribe-block__organizer__details {
+		h3, p {
+			font-size: 1.65rem;
+		}
+	}
+}
```
Comment (paulmskim) [inline src/styles/event-organizer/frontend.pcss:74]:
```suggestion
/* Twenty-twenty theme */
.theme-twentytwenty {

	.tribe-block__organizer__details {

		h3,
		p {
			font-size: 1.65rem;	
		}
	}
}

/* Divi */
.theme-Divi {

	.tribe-block__organizer__details {

		h3,
		p {
			padding-bottom: 0;		
		}
	}
}

/* Genesis */
.theme-genesis {

	.tribe-block__organizer__details {

		h3,
		p {
			font-size: 1.65rem;
		}
	}
}
```

File: src/styles/event-organizer/frontend.pcss [hunk 8]
```diff
@@ -4,22 +4,85 @@
...
+
+/* Avada */
+.theme-Avada .tribe-block__organizer__details {
+	max-width: 80%;
+
+	@media screen and (min-width: 768px) {
+		max-width: 40%;
+	}
+}
```
Comment (paulmskim) [inline src/styles/event-organizer/frontend.pcss:88]: left a note in slack about reaching out to avada about overrides.

**Comment by EugeneKyale** [comment]
@paulmskim lets wait and hear what the team has to say about this.

File: src/styles/event-organizer/frontend.pcss (hunk 8 above)
Comment (EugeneKyale) [reply src/styles/event-organizer/frontend.pcss:88]: lets wait and hear what the team has to say about this.

File: src/resources/postcss/tribe-events-full.pcss (hunk 1 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-full.pcss]: @PaulHughes01 I've removed the override targeting Enfold.

**Comment by EugeneKyale** [comment]
This PR is very dependent on this [one](https://github.com/the-events-calendar/the-events-calendar/pull/3614). As a result of this I'll fix the failing tests one the former is merged.

File: src/resources/postcss/tribe-events-single-blocks-reskin.pcss [hunk 9]
```diff
@@ -122,15 +105,59 @@
 	}
 }
 
-/* Theme specific overrides */
+/* = Organizer block view styles
+============================================= */
+.tribe-block__organizer__details {
+	border-top: 1px solid #d5d5d5;
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks-reskin.pcss:111]: are we using variables or hardcoded colors?

File: src/resources/postcss/tribe-events-single-blocks-reskin.pcss [hunk 10]
```diff
@@ -160,51 +196,70 @@
...
+		h3,
+		p {
+			padding-bottom: 0;
+		}
+	}
 }
 
-/* Avada */
+/* Avada theme */
 .tribe-theme-avada {
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks-reskin.pcss:259]: let's remove this section

**Review by paulmskim — CHANGES_REQUESTED** [review]

File: src/resources/postcss/tribe-events-single-blocks-reskin.pcss (hunk 9 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks-reskin.pcss]: I've added a variable here

File: src/resources/postcss/tribe-events-single-blocks-reskin.pcss (hunk 10 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks-reskin.pcss]: I've removed the overrides for Avada

File: src/resources/postcss/tribe-events-single-blocks-reskin.pcss (hunk 9 above)
Comment (paulmskim) [reply src/resources/postcss/tribe-events-single-blocks-reskin.pcss]: there are more colors that need to be changed to variables in this PR

File: src/resources/postcss/tribe-events-single-blocks.pcss [hunk 11]
```diff
@@ -118,15 +101,59 @@
...
 /* Twenty-twenty theme */
 .tribe-theme-twentytwenty {
+	/* Website block view styles */
+	.tribe-block__event-website a {
+		font-size: 1.65rem;
+	}
+
+	/* Venue block view styles */
 	.tribe-block__venue {
 		.tribe-block__venue__meta {
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks.pcss:158]: combine these two into one line, no needed to nest unnecessarily

File: src/resources/postcss/tribe-events-single-blocks.pcss [hunk 12]
```diff
@@ -142,10 +169,19 @@
...
+		}
+	}
 }
 
-/* Twenty Seventeen */
+/* Twenty-seventeen theme */
 .tribe-theme-twentyseventeen {
+	/* Venue block view styles */
 	.tribe-block__venue {
 		.tribe-block__venue__meta {
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks.pcss:186]: combine these to one line

File: src/resources/postcss/tribe-events-single-blocks.pcss [hunk 13]
```diff
@@ -156,44 +192,68 @@
...
+/* Genesis theme */
+.tribe-theme-genesis {
+	/* Website block view styles */
+	.tribe-block__event-website a {
+		font-size: 1.65rem;
+	}
+
+	/* Venue block view styles */
 	.tribe-block__venue {
 		.tribe-block__venue__meta {
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks.pcss:204]: combine to one line

File: src/resources/postcss/tribe-events-single-blocks.pcss [hunk 14]
```diff
@@ -156,44 +192,68 @@
...
+	}
 }
 
-/* Genesis */
-.tribe-theme-genesis {
+/* Divi theme */
+.tribe-theme-divi {
+	/* Venue block view styles */
 	.tribe-block__venue {
 		.tribe-block__venue__meta {
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks.pcss:231]: combine to one line

File: src/resources/postcss/tribe-events-single-blocks.pcss [hunk 15]
```diff
@@ -156,44 +192,68 @@
...
+			padding-bottom: 0;
+		}
+	}
 }
 
 /* Enfold */
 .tribe-theme-enfold {
+	/* Venue block view styles */
 	.tribe-block__venue {
 		.tribe-block__venue__meta {
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks.pcss:258]: combine to one line

**Review by paulmskim — CHANGES_REQUESTED** [review]

File: src/resources/postcss/tribe-events-single-blocks.pcss (hunk 15 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks.pcss]: done.

File: src/resources/postcss/tribe-events-single-blocks.pcss (hunk 14 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks.pcss]: done.

File: src/resources/postcss/tribe-events-single-blocks.pcss (hunk 13 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks.pcss]: done.

File: src/resources/postcss/tribe-events-single-blocks.pcss (hunk 12 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks.pcss]: done.

File: src/resources/postcss/tribe-events-single-blocks.pcss (hunk 11 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks.pcss]: done.

File: src/resources/postcss/tribe-events-single-blocks-reskin.pcss (hunk 9 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks-reskin.pcss]: I've updated all the colors now

**Review by paulmskim — APPROVED** [review]

**Pull request merged**